invenio_sequencegenerator/admin.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015, 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Admin views for invenio-accounts."""
+
+from flask import url_for, redirect, flash
+from flask_admin.actions import action
+from flask_admin.contrib.sqla import ModelView
+from flask_babelex import gettext as _, ngettext
+from invenio_db import db
+
+from .models import Counter, TemplateDefinition
+
+
+class TemplateDefinitionView(ModelView):
+    """Flask-Admin view for template definitions."""
+
+    column_list = (
+        'name',
+        'meta_template',
+        'parent_name',
+        'start',
+        'step',
+    )
+
+
+class CounterView(ModelView):
+    """Admin view for counters."""
+
+    column_list = (
+        'template_instance',
+        'counter',
+        'definition_name',
+    )
+
+    @action('reset', 'Reset',
+            'Are you sure you want to reset selected counter?')
+    def reset(self, ids):
+        """Reset selected counter."""
+        for id in ids:
```

Comment:
id is a function of python you should try to avoid to use variable with the same name, even if it doesn't break anything here.